test/SerilogWeb.Classic.Tests/WebRequestLoggingHandlerTests.cs

```diff
@@ -327,6 +327,43 @@ public void PasswordFilteringCanBeDisabled()
             Assert.Equal(expectedLoggedData.ToString(), formDataProperty.ToString());
         }
 
+        [Fact]
+        public void ErrorIfMethodFilterKeywordsCalledWithNullCollection()
+        {
+            Type expectedExceptionType = typeof(ArgumentNullException);
+            string expectedExceptionMessage = "Value cannot be null.\r\nParameter name: keywordBlackList";
```

Comment:
nitpicking : maybe we don't need separate variables to store those and can do directly : 

```cs
Assert.IsType(typeof(ArgumentNullException), actualException);
Assert.Equal("Value cannot be null.\r\nParameter name: keywordBlackList", actualException.Message);
```

or at least move the variable declaration closer to where it is used ? 

(this means you don't need to scroll up to the top of the test to know what those values are when reading the assertions , and also when reading the test top down, you have less clutter until you reach the actual "Act" part of the test)

Reply:
This was my first approach and I see it as being the better one now. That variable doesn't contribute as much to "expectancy" clarity as I believed. Will change.